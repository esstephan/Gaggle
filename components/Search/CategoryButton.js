import React from 'react';
import { Button, Text } from 'native-base';

const CategoryButton = props => (
  <Button
    block
    success
    style={{ marginLeft: 5, marginRight: 5, marginTop: 5, height: 30 }}
    onPress={() => props.getSuggestions(props.category)}
  >
    <Text>{props.category}</Text>
  </Button>
);

CategoryButton.propTypes = {
  category: React.PropTypes.string.isRequired,
  getSuggestions: React.PropTypes.func.isRequired,
};

export default CategoryButton;
