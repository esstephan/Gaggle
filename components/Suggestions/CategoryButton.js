import React from 'react';
import { Button, Text } from 'native-base';

const CategoryButton = props => (
  <Button onPress={() => {}}>
    <Text>{props.category}</Text>
  </Button>
);

CategoryButton.propTypes = {
  category: React.PropTypes.string.isRequired,
};

export default CategoryButton;
