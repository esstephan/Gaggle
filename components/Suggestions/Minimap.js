import React, { Component } from 'react';
import MapView from 'react-native-maps';
import { getUserLocation } from '../../location/locationHelpers';

export default class Minimap extends Component {
  constructor(props) {
    super(props);
    this.state = {
      myLocation: [0, 0],
    };
    getUserLocation().then((myLocation) => {
      this.setState({ myLocation });
    });
  }

  componentDidUpdate() {
    // Hacky way to animate to fitToElements
    setTimeout(() => this.refs.map.fitToElements(true), 1500);
  }

  render() {
    return (
      <MapView
        ref='map'
        style={{ height: 150, width: 375 }}
        region={{
          latitude: this.props.coords[0],
          longitude: this.props.coords[1],
          latitudeDelta: 0.1,
          longitudeDelta: 0.1,
        }}
        showsUserLocation
      >
        <MapView.Marker
          coordinate={{ latitude: this.props.coords[0], longitude: this.props.coords[1] }}
          title={'Some place'}
        />
      </MapView>
    );
  }
}

Minimap.propTypes = {
  coords: React.PropTypes.array.isRequired,
};
