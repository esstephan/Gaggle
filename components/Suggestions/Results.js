import React from 'react';
import { List, ListItem, Text } from 'native-base';

const Results = (props) => {
  const goToResultDetails = (details) => {
    console.log(details);
    console.log(props);
  };

  return (
    <List>
      {props.results.map(result => (
        <ListItem
          key={result.id}
          onPress={() => goToResultDetails(result)}
        >
          <Text>
            {result.name}{'\n'}
            [{result.geometry.location.lat}, {result.geometry.location.lng}]{'\n'}
            {result.vicinity}
          </Text>
        </ListItem>
      ))}
    </List>
  );
};

Results.propTypes = {
  results: React.PropTypes.array.isRequired,
};

export default Results;
