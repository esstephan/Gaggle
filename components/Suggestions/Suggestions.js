import React, { Component } from 'react';
import { Container, Header, Content, Text } from 'native-base';

export default class Suggestions extends Component {
  constructor(props) {
    super(props);
    this.state = {

    };
  }

  render() {
    return (
      <Container>
        <Header />
        <Content>
          <Text>Hello World</Text>
        </Content>
      </Container>
    );
  }
}
