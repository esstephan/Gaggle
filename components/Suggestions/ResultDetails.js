import React, { Component } from 'react';
import { View } from 'react-native';
import { Container, Header, Content, Text } from 'native-base';
// import { Grid, Col } from 'react-native-easy-grid';
import Autolink from 'react-native-autolink';
import moment from 'moment';
import { getPlaceDetails } from '../../google/googlePlaces';
import Stars from './Stars';
import Minimap from './Minimap';

export default class ResultDetails extends Component {
  constructor(props) {
    super(props);
    this.state = {
      place: {},
      placeLocation: [0, 0],
    };

    getPlaceDetails(this.props.placeId).then((data) => {
      console.log('Getting result details: ', data.result);
      const place = data.result;
      this.setState({
        place,
        placeLocation: [place.geometry.location.lat, place.geometry.location.lng],
      });
    });
  }

  render() {
    const place = this.state.place;

    return (
      <Container>
        <Header />
        <Content>
          <Minimap coords={this.state.placeLocation} placeName={place.name || 'Waiting for place name...'} />
          <Text>{place.name}</Text>
          <Text>{place.formatted_address}</Text>
          {place.international_phone_number ?
            <Autolink text={place.international_phone_number} /> : null}
          {place.website ? <Autolink text={place.website} /> : null}
          {place.opening_hours && <Text>Open Now: {place.opening_hours.open_now ? 'Yes' : 'No'}</Text>}
          <Stars stars={Math.floor(place.rating)} />
          {place.reviews && <Text>Reviews:</Text>}
          {place.reviews ? place.reviews.map(review =>
            <View key={review.author_name}>
              <Text>{review.author_name} ({moment.unix(review.time).fromNow()}) :</Text>
              <Stars stars={Math.floor(review.rating)} />
              <Text>{review.text}</Text>
            </View>,
            ) : null}
        </Content>
      </Container>
    );
  }
}

ResultDetails.propTypes = {
  placeId: React.PropTypes.string.isRequired,
};
