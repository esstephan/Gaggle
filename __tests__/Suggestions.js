import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Suggestions from '../components/Suggestions/Suggestions';
import CategoryButton from '../components/Suggestions/CategoryButton';

describe('Suggestions main component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Suggestions
        groupName={'Default'}
      />,
    );
    expect(tree).toMatchSnapshot();
  });
});

describe('CategoryButton', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <CategoryButton category={'Fun'} getSuggestions={() => {}} />,
    );
    expect(tree).toMatchSnapshot();
  });
});
