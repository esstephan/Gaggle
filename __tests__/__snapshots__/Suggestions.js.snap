// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryButton renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "backgroundColor": "#5cb85c",
      "borderBottomWidth": null,
      "borderColor": "#007aff",
      "borderLeftWidth": null,
      "borderRadius": 5,
      "borderRightWidth": null,
      "borderTopWidth": null,
      "elevation": 2,
      "flexDirection": "row",
      "height": 30,
      "justifyContent": "center",
      "marginBottom": 5,
      "marginLeft": 5,
      "marginRight": 5,
      "marginTop": 5,
      "opacity": 1,
      "paddingBottom": 6,
      "paddingLeft": 16,
      "paddingRight": 16,
      "paddingTop": 6,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View
    style={
      Object {
        "alignSelf": "center",
        "zIndex": 999,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#fff",
        "fontFamily": "System",
        "fontSize": 16.5,
        "lineHeight": 19,
        "marginLeft": 0,
        "marginRight": 0,
      }
    }
    virtual={undefined}
  >
    Fun
  </Text>
</View>
`;

exports[`Suggestions main component renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "height": 1334,
    }
  }
  virtual={undefined}
>
  <View>
    <View
      style={
        Object {
          "backgroundColor": "#F8F8F8",
          "borderBottomColor": "#a7a6ab",
          "borderBottomWidth": 0.5,
          "elevation": 3,
          "flexDirection": "row",
          "height": 64,
          "justifyContent": "center",
          "left": 0,
          "paddingLeft": 10,
          "paddingRight": 10,
          "paddingTop": 15,
          "right": 0,
          "shadowColor": undefined,
          "shadowOffset": undefined,
          "shadowOpacity": undefined,
          "shadowRadius": undefined,
          "top": 0,
        }
      }
      virtual={undefined}
    />
  </View>
  <RCTScrollView>
    <View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Restaurants
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Museums
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Cafes
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Shopping
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Bars
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Movies
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Games
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Recreation
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Art
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Music
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Shows
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Theaters
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "elevation": 2,
            "flexDirection": "row",
            "height": 30,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 5,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 6,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "zIndex": 999,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontFamily": "System",
              "fontSize": 16.5,
              "lineHeight": 19,
              "marginLeft": 0,
              "marginRight": 0,
            }
          }
          virtual={undefined}
        >
          Skydiving
        </Text>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#000",
            "fontFamily": "System",
            "fontSize": 16,
          }
        }
        virtual={undefined}
      >
        Default
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#000",
            "fontFamily": "System",
            "fontSize": 16,
          }
        }
        virtual={undefined}
      >
        []
      </Text>
      <View
        style={Object {}}
        virtual={undefined}
      />
    </View>
  </RCTScrollView>
</View>
`;
